trie/database.go

```diff
@@ -816,6 +817,60 @@ func (db *Database) Update(nodes *MergedNodeSet) error {
 	return nil
 }
 
+func (db *Database) SetCacheNodeSet(set *NodeSet) error {
+	if db.cacheMergedNodeSet == nil {
+		db.cacheMergedNodeSet = NewMergedNodeSet()
+	}
+	return db.cacheMergedNodeSet.Merge(set)
+}
+
+func (db *Database) UpdateForOKC(accRetrieval func([]byte) common.Hash) error {
```

Comment:
```suggestion
func (db *Database) OrgnizeTree(accRetrieval func([]byte) common.Hash) error {
```

Reply:
This method Name is copy for method 'Update' ,so i use 'UpdateForOKC'